tools/pot/openvino/tools/pot/graph/node_utils.py

```diff
@@ -267,6 +267,19 @@ def get_lstm_ends(read_value, assigns, ignore_nodes):
     return lstm_outputs
 
 
+def create_node_name(input_node, mode=tuple):
+    """
+    Returns key for node input.
+    If input node has one output port -> key is name of input node.
+    Otherwise, key is tuple (input name, output port number)
+    """
+    key = input_node.fullname
```

Comment:
There is the almost same method exists: `node_utils.get_quantized_input_key`. Maybe it will be good to extend existing method or update it with the new one.

Reply:
Thanks. Will be updated.